src/App.js

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import Header from './modules/Header';
+import Home from './modules/Home';
+import BookList from './modules/Routes/BookList';
+import Categories from './modules/Routes/Categories';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/books" element={<BookList />} />
+        <Route path="/categories" element={<Categories />} />
```

Comment:
✅ Has 2 `<Route>`s, one for the BookList component and the other for the Categories component 👍🏻 

- [ ] [REQUIRED] Kindly make the BookList component as the default page as requested in the project requirements